Source/WebCore/rendering/BreakLines.h

```diff
@@ -29,45 +29,65 @@
 
 namespace WebCore {
 
-enum class NoBreakSpaceBehavior {
-    Normal,
-    Break,
-};
-
-enum class WordBreakBehavior {
-    Normal,
-    BreakAll,
-    KeepAll,
-};
+class BreakLines {
+public:
+    enum class NoBreakSpaceBehavior {
```

Comment:
nit: maybe use this cleanup to set the underlying type of these scoped enums (normal WebKit practice)?

Reply:
I could set them to something, but since they are only used for templating it actually doesn't matter what they map to... seems like it would just be unnecessary noise?